src/main/osd/osd_elements.c

```diff
@@ -2094,81 +2168,131 @@ uint8_t osdGetActiveElementCount(void)
-            memset(buff,0,strlen(buff));
-            len = tfp_sprintf(buff, "KV %d   POLES %d", motorConfig()->kv, motorConfig()->motorPoleCount);
-            displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
+        specState = MIXER;
+        break;
 
-            memset(buff,0,strlen(buff));
-            len = tfp_sprintf(buff, "%d  %d  %d", mixerConfig()->rpm_limit_p, mixerConfig()->rpm_limit_i, mixerConfig()->rpm_limit_d);
-            displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
-        } else
+    case MIXER:
+        memset(buff,0,strlen(buff));
+        len = tfp_sprintf(buff, "%d  %d  %d", mixerConfig()->rpm_limit_p, mixerConfig()->rpm_limit_i, mixerConfig()->rpm_limit_d);
+        displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
+
+        specState = THR;
+        break;
+
+    case THR:
 #endif // #USE_RPM_LIMIT
-        {
-            memset(buff,0,strlen(buff));
-            len = tfp_sprintf(buff, "THR LIMIT %s", lookupTableThrottleLimitType[currentControlRateProfile->throttle_limit_type]);
-            if (currentControlRateProfile->throttle_limit_type != THROTTLE_LIMIT_TYPE_OFF) {
-                len = tfp_sprintf(buff, "%s %d", buff, currentControlRateProfile->throttle_limit_percent);
-            }
-            displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
+        memset(buff,0,strlen(buff));
+        len = tfp_sprintf(buff, "THR LIMIT %s", lookupTableThrottleLimitType[currentControlRateProfile->throttle_limit_type]);
+        if (currentControlRateProfile->throttle_limit_type != THROTTLE_LIMIT_TYPE_OFF) {
+            len = tfp_sprintf(buff, "%s %d", buff, currentControlRateProfile->throttle_limit_percent);
         }
+        displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
 
+        specState = MOTOR;
+        break;
+
+    case MOTOR:
         memset(buff,0,strlen(buff));
         len = tfp_sprintf(buff, "MOTOR LIMIT %d", currentPidProfile->motor_output_limit);
         displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
 
+        specState = BAT;
+        break;
+
+    case BAT:
         memset(buff,0,strlen(buff));
         const float batteryVoltage = getBatteryVoltage() / 100.0f;
         len = osdPrintFloat(buff, osdGetBatterySymbol(getBatteryAverageCellVoltage()), batteryVoltage, "", 2, true, SYM_VOLT);
         displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, buff);
 
+        specState = VER;
+        break;
+
+    case VER:
         len = strlen(FC_VERSION_STRING);
-        displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, FC_VERSION_STRING);
+        displayWrite(osdDisplayPort, midCol - len/2, currentRow, DISPLAYPORT_SEVERITY_NORMAL, FC_VERSION_STRING);
```

Comment:
```suggestion
        displayWrite(osdDisplayPort, midCol - len/2, currentRow++, DISPLAYPORT_SEVERITY_NORMAL, FC_VERSION_STRING);
```

If possible, keep all `case` similar

Reply:
done